graylog2-rest-models/src/main/java/org/graylog2/rest/models/alarmcallbacks/AlarmCallbackError.java

```diff
@@ -32,6 +33,6 @@
 
     @JsonCreator
     public static AlarmCallbackError create(@JsonProperty("error") String error) {
-        return new AutoValue_AlarmCallbackError(error);
+        return new AutoValue_AlarmCallbackError(Strings.nullToEmpty(error));
```

Comment:
Why is this necessary? The error message shouldn't be `null` (or empty for that matter) but mention the reason for the error.